incubator/kafka/templates/service-brokers.yaml

```diff
@@ -18,19 +18,19 @@ spec:
     targetPort: {{ .Values.prometheus.jmx.port }}
 {{- end }}
   selector:
-    app: {{ include "kafka.name" . | quote }}
-    release: {{ .Release.Name | quote }}
+    app: {{ include "kafka.name" . }}
+    release: {{ .Release.Name }}
 ---
 {{- if and .Values.prometheus.kafka.enabled .Values.prometheus.operator.enabled }}
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ template "kafka.fullname" . }}-exporter
+  name: {{ include "kafka.fullname" . }}-exporter
```

Comment:
e.g. updating chart to consistently use include over template (rather than intermix the two), as described here:

https://github.com/kubernetes/helm/blob/master/docs/chart_template_guide/named_templates.md

Reply:
👍